docs/content/how-to/aws/migrate-hosted-cluster.md

```diff
@@ -0,0 +1,990 @@
+---
+title: Migrating Hosted Cluster among the same AWS Region
+---
+
+# Migrating Hosted Cluster among the same AWS Region
+
```

Comment:
Can we elaborate a bit that this intents to be a migration transparent for the guest cluster data plane with no user workloads disruption where providing Node instances are always running during the process.